lib/patches/active_record/xml_attribute_serializer.rb

```diff
@@ -1,13 +1,18 @@
 require 'active_record/serializers/xml_serializer'
```

Comment:
It lives here for now https://github.com/rails/activemodel-serializers-xml/blob/master/lib/active_record/serializers/xml_serializer.rb.